public/js/plugins/bootstrap-input-spinner.js

```
@@ -7,327 +7,328 @@
           }
-          value = NaN;
-        } else {
-          newValue = parseFloat(newValue);
-          newValue = Math.min(Math.max(newValue, min), max);
-          newValue =
-            Math.round(newValue * Math.pow(10, decimals)) /
-            Math.pow(10, decimals);
-          $original[0].value = newValue;
-          if (updateInput) {
-            $input[0].value = numberFormat.format(newValue);
+          if (isNaN(newValue) || newValue === "") {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)